### PR TITLE
New options --html-medium-thereshold and --html-high-threshold (:issu…

### PR DESCRIPTION
**Pull request by Spacetown**
…e:#261)
New request after splitting #261 and problems with the repo.

### CODE REVIEW

File: gcovr/html_generator.py [hunk 1]
```diff
@@ -88,11 +85,13 @@ def coverage_to_color(coverage):
...
 #
 def print_html_report(covdata, options):
+    medium_threshold = options.html_medium_threshold
+    high_threshold = options.html_high_threshold
     details = options.html_details
     if options.output is None:
         details = False
     data = {}
-    data['HEAD'] = options.html_title
+    data['HEAD'] = options.title
```
Comment (latk) [inline gcovr/html_generator.py]: I think the tests are failing because of this line, otherwise this looks great!

**Comment by latk** [comment]
Thank you, this looks very good! I'll take a closer look tomorrow and merge.

I'm sorry that you had problems with Git, it can sometimes be totally confusing!

**Comment by Spacetown** [comment]
I'm coming from IBM Rational ClearCase, that is also confusing. I'm working the first time with GitHub.

File: gcovr/html_generator.py (hunk 1 above)
Comment (Spacetown) [reply gcovr/html_generator.py]: Already pushed. How do I start the test again?

Comment (latk) [reply gcovr/html_generator.py]: The tests restart automatically when you push. The remaining build failures are not test failures (wohoo!), but just style guide violations, see the build log: <https://travis-ci.org/gcovr/gcovr/jobs/392858479#L925>. To run the style checker locally, install flake8 from PyPI.

Comment (Spacetown) [reply gcovr/html_generator.py]: Ok. At work I'm using SmartGit under Windows and at home I'm working with different tools under MacOs the first time.

**Comment by latk** [comment]
Thank you for your contribution, it's merged now! I squashed the commits and touched up minor typos, which is why I had to force-push to this branch.

**Pull request merged**